Error when sending hex color string to color item

In my production environment I get the following error message when doing the following command:

```
 led << '#FF0000'
```

in which led is a Color item. 

This is the offending line of code
```
  QuantityType.new(raw_hue.to_big_decimal, ::Units::DEGREE_ANGLE)
```

```
2021-10-29 16:39:59.368 [ERROR] [rulesupport.shared.simple.SimpleRule] - org.openhab.core.library.unit.Units is not a type/class (TypeError)
In rule: Turn on motion LED
/etc/openhab/automation/lib/ruby/gem_home/gems/openhab-scripting-4.8.0/lib/openhab/dsl/types/hsb_type.rb:153:in `hue'
/etc/openhab/automation/lib/ruby/gem_home/gems/openhab-scripting-4.8.0/lib/openhab/dsl/types/hsb_type.rb:178:in `to_s'
/etc/openhab/automation/lib/ruby/gem_home/gems/openhab-scripting-4.8.0/lib/openhab/dsl/items/generic_item.rb:59:in `command'
/etc/openhab/automation/lib/ruby/gem_home/gems/openhab-scripting-4.8.0/lib/openhab/dsl/items/ensure.rb:55:in `command'
/etc/openhab/automation/lib/ruby/gem_home/gems/openhab-scripting-4.8.0/lib/openhab/dsl/items/timed_command.rb:47:in `command'
/etc/openhab/automation/jsr223/ruby/personal/esp_home.rb:31:in `block in <main>'
```

We have a test for this specific thing which passes.. So I am confused why it is failing in production and also why it woudln't be able to find Units.

--- a/lib/openhab/dsl/types/hsb_type.rb
+++ b/lib/openhab/dsl/types/hsb_type.rb
@@ -78,7 +78,7 @@ module OpenHAB
             args[0] = if hue.is_a?(DecimalType)
                         hue
                       elsif hue.is_a?(QuantityType)
-                        DecimalType.new(hue.to_unit(::Units::DEGREE_ANGLE).to_big_decimal)
+                        DecimalType.new(hue.to_unit(org.openhab.core.library.unit.Units::DEGREE_ANGLE).to_big_decimal)
                       elsif hue.respond_to?(:to_d)
                         DecimalType.new(hue)
                       end
@@ -150,7 +150,7 @@ module OpenHAB
         # @!attribute [r] hue
         # @return [QuantityType]
         def hue
-          QuantityType.new(raw_hue.to_big_decimal, ::Units::DEGREE_ANGLE)
+          QuantityType.new(raw_hue.to_big_decimal, org.openhab.core.library.unit.Units::DEGREE_ANGLE)
         end
 
         # Convert to a packed 32-bit RGB value representing the color in the default sRGB color model.

